Geometry/HGCalCommonData/src/HGCalGeomRotation.cc

```diff
@@ -135,8 +133,7 @@ unsigned HGCalGeomRotation::uvMappingTo60DegreeSector0(WaferCentring waferCentri
     } else {
       sector = 1;
     }
-  }
-  if (moduleU >= moduleV && moduleV < 0) {
+  } else if (moduleU >= moduleV && moduleV < 0) {
```

Comment:
Once the PR has been finalized, the updates/fixes in `HGCalCommonData` should be PRed into central CMSSW

Reply:
I was already asked to do this (which is how I found out about the bug) - the PR is here: https://github.com/cms-sw/cmssw/pull/33191